apps/frontend/src/pages/Home.tsx

```diff
@@ -6,12 +6,16 @@ import useHabits, { Habit } from "../util/hooks/useHabits";
 import CreateHabitDialog, { HabitForm } from "../components/CreateHabitDialog";
 import APIRequest from "../util/request";
 import DeleteHabitDialog from "../components/DeleteHabitDialog";
+import ProgressHabitDialog from "../components/ProgressHabitDialog";
 
 export default function Home() {
   const [openDelete, setOpenDelete] = useState(false);
   const [habitToDelete, setHabitToDelete] = useState<Habit>();
   const [openCreate, setOpenCreate] = useState(false);
   const [habits, addHabits, deleteHabits] = useHabits([]);
+  //Progress consts VVV
+  const [habitToProgress, setHabitToProgress] = useState<Habit>();
+  const [openProgress, setOpenProgress] = useState(false);
```

Comment:
I think these names will also conflict with the habit-progress merge from yesterday